### PR TITLE
Adding Resources for Android Developers

### PR DESCRIPTION
**Pull request by Akashsri3bi**
@shahednasser  Adding Study Guide , Exam Link for Android Developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Android Developer Roadmap](https://roadmap.sh/android)
 - [Android cheat sheet](http://alvinalexander.com/android/android-cheat-sheet-main-concepts/)
 - [Android Tips & Tricks](https://github.com/nisrulz/android-tips-tricks)
+- [Android Developer Certification Exam Guide](https://developers.google.com/certification/associate-android-developer/study-guide)
```
Comment (shahednasser) [inline README.md:79]: Replace with the actual title of the page.

File: README.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [Android developer Exam](https://app.trueability.com/google-certifications/associate-android-developer)
```
Comment (shahednasser) [inline README.md:80]: I'm not sure how valid this is so please remove it.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @Akashsri3bi for your contribution. I just have a few comments below.

File: README.md (hunk 1 above)
Comment (Akashsri3bi) [reply README.md:79]: @shahednasser this is valid you can checkout the link for confirmation

Comment (shahednasser) [reply README.md:79]: I think you mean the link below. I checked the page, but I still prefer to remove it.

Comment (Akashsri3bi) [reply README.md:79]: @shahednasser can I atleast know the reason why it cannot be merged ?

**Pull request closed without merging**